src/Spryker/Zed/AppPayment/Business/AppPaymentFacadeInterface.php

```diff
@@ -228,4 +231,24 @@ public function confirmPreOrderPayment(
     public function cancelPreOrderPayment(
         CancelPreOrderPaymentRequestTransfer $cancelPreOrderPaymentRequestTransfer
     ): CancelPreOrderPaymentResponseTransfer;
+
+    /**
+     * Specification:
+     * - Loads the payment method by the tenant identifier and payment method key.
+     * - Returns a PaymentMethodTransfer
+     *
+     * @api
+     */
+    public function getPaymentMethodByTenantIdentifierAndPaymentMethodKey(string $tenantIdentifier, string $paymentMethodKey): PaymentMethodTransfer;
```

Comment:
We often provide DTOs for facades to allow easy method extension. In this case, `getPaymentMethod(PaymentMethodCriteriaTransfer)` would be a better approach.